Source/Core/DolphinWX/Main.cpp

```diff
@@ -208,9 +208,6 @@ void DolphinApp::MacOpenFile(const wxString& fileName)
 
 void DolphinApp::AfterInit()
 {
-  if (!m_batch_mode)
-    main_frame->UpdateGameList();
```

Comment:
Not rescanning the game list when opening Dolphin is a pretty big regression in UX. Is there any particular reason for it? Users will have a harder time adding new games (will they realize that they have to use the refresh button?), and having deleted games stick around seems pretty buggy considering that Dolphin will crash when you try to open the ISOProperties for a deleted game.